src/ppx/application.cpp

```diff
@@ -1400,6 +1398,16 @@ void Application::DrawDebugInfo(std::function<void(void)> drawAdditionalFn)
     }
     uint32_t minWidth  = std::min(kImGuiMinWidth, GetWindowWidth() / 2);
     uint32_t minHeight = std::min(kImGuiMinHeight, GetWindowHeight() / 2);
+#if defined(PPX_BUILD_XR)
+    // For XR, force the diagnostic window to the center with automatic sizing for legibility and since control is limited.
+    static ImVec2 lastWindowSize(0.f, 0.f);
+    ImGui::SetNextWindowPos({(GetWindowWidth() - lastWindowSize.x) / 2, (GetWindowHeight() - lastWindowSize.y) / 2}, 0, {0.0f, 0.0f});
+    ImGui::SetNextWindowSize({0, 0});
+#else
+    // For Non-XR, spawn the window in the upper left region, but allow it to be resized and repositioned.
+    // Note that the FirstUseEver conditional means any value from previous runs (stored in imgui.ini) will be used if available.
+    ImGui::SetNextWindowPos({10.f, 10.f}, ImGuiCond_FirstUseEver, {0.0f, 0.0f});
```

Comment:
What is the default behaviour here if we don't set this? It looks like it is already in the upper left region without this code, do we need to set anything here?